pymatbridge/tests/test_magic.py

```diff
@@ -82,7 +82,7 @@ def test_matrix(self):
     # Matlab struct type should be converted to a Python dict
     def test_struct(self):
         self.ip.run_cell('num = 2.567')
-        self.ip.run_cell('num_array = np.array([1.2,3.4,5.6])')
+        self.ip.run_cell('num_array = np.array([1.2,3.4,5.6]).reshape(3,1)')
```

Comment:
I fixed it so it sends as a row vector, but it would still fail based on (1,3) != (3,).  I believe that vectors will be the only place we run into this.   We could look for a shape of `(1,n)` coming back and squeeze it, which should have wider applicability, but still would not be perfect.